### PR TITLE
Install scalafmt and reformat all source

### PR DESCRIPTION
**Pull request by sihil**
Tackle #335.

Whilst I've installed scalafmt as a plugin it is not wired up to run automatically as my reading of #335 indicates that @philwills would be disappointed if this were the case.

I'm happy to remove the plugin entirely if that is deemed better.

### CODE REVIEW

**Comment by philwills** [comment]
I'm fairly convinced that a fair chunk of this is less readable, but I'm willing to be overruled.

File: magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala [hunk 1]
```diff
@@ -11,44 +11,48 @@ object AmiCloudFormationParameter extends DeploymentType {
...
-  val amiTags = Param[Map[String,String]]("amiTags",
-    documentation = "Specify the set of tags to use to find the latest AMI"
-  )
-  val amiParameter = Param[String]("amiParameter",
-    documentation = "The CloudFormation parameter name for the AMI"
-  ).default("AMI")
+  val cloudFormationStackName =
+    Param[String]("cloudFormationStackName", documentation = "The name of the CloudFormation stack to update")
+      .defaultFromPackage(_.name)
+  val prependStackToCloudFormationStackName = Param[Boolean](
```
Comment (philwills) [inline magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala]: I really don't believe 5 lines is better than 3 here.

Comment (sihil) [inline magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala]: Fair point.

**Comment by sihil** [comment]
I'm happy to close this, but thought I'd give it a bash in a moment of enthusiasm.

**Comment by philwills** [comment]
Maybe it's worth trying this and then going back an fixing up things we don't like afterwards.  It's always easy to focus on the bad things in a change rather than the good and it may well be that the balance is in favour of this.

**Pull request closed without merging**